db/migrate/20210222102503_create_users_settings.rb

```diff
@@ -0,0 +1,25 @@
+class CreateUsersSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_settings do |t|
+      t.integer :user_id, null: false
```

Comment:
should this be using `t.references(foreign_key: true)` https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-references ? This way we add a DB level foreign key as well.

Otherwise `user_id` could be any integer value

Reply:
✅